packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.tsx

```diff
@@ -115,6 +115,10 @@ export class SearchBox extends BaseComponent<ISearchBoxProps, ISearchBoxState> {
     });
 
     this._events.on(this._rootElement, 'blur', this._onBlur, true);
+
+    if (this.props.onFocus) {
+      this.props.onFocus(ev as React.FocusEvent<HTMLInputElement>);
```

Comment:
ISearchBoxProps extends React.InputHTMLAttributes<HTMLInputElement>, so this.props.onFocus are for HTMLInputElement and therefore I had to cast this to avoid compiler error.